### ISSUE TITLE
MaterialX: Unlit materials (single-output nodegraphs) don't work

### ISSUE THREAD

**Issue by hybridherbst** [mrdoob/three.js#27494]
### Description

Seems that nodegraphs that don't use a surface model (aka: are unlit) are not supported right now, e.g. this one doesn't work:

```xml
<?xml version="1.0"?>
<materialx version="1.38" colorspace="lin_rec709">
  <nodegraph name="just_unlit">
    <mix name="mix_color3" type="color3">
      <input name="fg" type="color3" value="0, 0.0986187, 0.186275" />
      <input name="bg" type="color3" value="0.735294, 0.735294, 0.735294" />
      <input name="mix" type="float" value="0.6" />
    </mix>
    <output name="out" type="color3" nodename="mix_color3" />
  </nodegraph>
</materialx>
```

The above works fine in https://academysoftwarefoundation.github.io/MaterialX/.

I think the relevant spec section is [this](https://github.com/AcademySoftwareFoundation/MaterialX/blob/d4c6f60984b9ec4d7be69d8402b97b0d18e63591/documents/Specification/MaterialX.Specification.md?plain=1#L1994), saying that if there's only a single output (and no surfacematerial) then that should be used, and it doesn't matter what its name is.

@sunag I think this should become a `MeshBasicNodeMaterial`, right?
Basically checking [here if any surfacematerial was found](https://github.com/mrdoob/three.js/blob/de6dd45d7e5aa58fed0fbc1dbe53def3402b39cc/examples/jsm/loaders/MaterialXLoader.js#L722), and if not, using the first output instead for an Unlit material.

### Reproduction steps

1. Load the above MaterialX with MaterialXLoader
2. Note it returns zero materials

(Side note: after discussion on the MaterialX slack there are additional cases – not really relevant for this issue, just interesting for the future. This file is also valid and contains two materials (one implicit unlit one and another explicit surfacematerial): [two-materials.mtlx.zip](https://github.com/mrdoob/three.js/files/13824498/two-materials.mtlx.zip))

**Comment by sunag** [comment]
> @sunagAcho que isso deveria se tornar um MeshBasicNodeMaterial, certo?

Yes, that would be the most appropriate.

Wouldn't it be more useful to expose the `nodegraphs` as we do with `materials`?

**Comment by hybridherbst** [comment]
That would potentially also be useful; but the main point why I opened this issue is that the nodegraph-only mtlx I posted above should return a single unlit material without me having to do anything extra.

**Comment by hybridherbst** [comment]
@sunag would it be possible for you to take a look at this issue? I'm not quite sure what the right approach would be. Would be great to see unlit MaterialX support.
